coverage/clover.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<coverage generated="1561735822284" clover="3.2.0">
+  <project timestamp="1561735822284" name="All files">
+    <metrics statements="2" coveredstatements="2" conditionals="6" coveredconditionals="6" methods="1" coveredmethods="1" elements="9" coveredelements="9" complexity="0" loc="2" ncloc="2" packages="1" files="1" classes="1"/>
+    <file name="utils.ts" path="/home/sam/shadowcat/src/utils/utils.ts">
+      <metrics statements="2" coveredstatements="2" conditionals="6" coveredconditionals="6" methods="1" coveredmethods="1"/>
+      <line num="2" count="1" type="stmt"/>
+      <line num="3" count="4" type="stmt"/>
+    </file>
+  </project>
+</coverage>
```

Comment:
I think the coverage directory needs to be in `.gitignore`?

Reply:
Oooh thanks! Good call.